chapters/envs.rst

```diff
@@ -14,157 +16,168 @@ Here is the list of environment variables that ``jina`` respects during runtime.
         .. highlight:: python
         .. code-block:: python
 
-            os.environ['JINA_LOG_PROFILING'] = 'true'
+            os.environ['JINA_ARRAY_QUANT'] = 'fp16'
 
             from jina.flow import Flow
 
 
+.. confval:: JINA_ARRAY_QUANT
 
-.. confval:: JINA_FULL_CLI
-
-    When set, show all Jina subcommands.
+    Quantization scheme when storing ndarray into protobuf message, useful for reducing the network latency and saving bandwidth. Possible values: ``fp16`` (almost lossless), ``uint8``.
 
     :default: unset
 
-.. confval:: JINA_PROFILING
-
-    Set to any non-empty value to turn on service-level time profiling for Jina.
-
-    :default: unset
+.. confval:: JINA_BINARY_DELIMITER
 
+    The delimiter used in :class:`BasePbIndexer`. Often use a delimiter phrase that you can determine does not occur. (Think of the mime message boundaries)
```

Comment:
what are the mime message boundaries

Reply:
that does not occur in any of the messages

Reply:
This is from this previous doc I didn't make any change. So to my understanding, this is just an example to explain delimiter, but this example may not be so good?

Reply:
not sure

Reply:
How about adding (e.g. B164240059B29C0E4EFEC397)